### ISSUE TITLE
undefined method `empty?' for nil:NilClass

### ISSUE THREAD

**Issue by dismine** [slonopotamus/asciidoctor-fb2#15]
```
undefined method `empty?' for nil:NilClass
  Use --trace for backtrace
```
```
asciidoctor-fb2 -v
Asciidoctor FB2 0.2.2 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
```

**Comment by slonopotamus** [comment]
I need output with `--trace`

**Comment by dismine** [comment]
```
asciidoctor-fb2 index.adoc --trace
Traceback (most recent call last):
        47: from /usr/local/bin/asciidoctor-fb2:23:in `<main>'
        46: from /usr/local/bin/asciidoctor-fb2:23:in `load'
        45: from /var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/bin/asciidoctor-fb2:24:in `<top (required)>'
        44: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/cli/invoker.rb:111:in `invoke!'
        43: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/cli/invoker.rb:111:in `each'
        42: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/cli/invoker.rb:128:in `block in invoke!'
        41: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/convert.rb:183:in `convert_file'
        40: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/convert.rb:183:in `open'
        39: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/convert.rb:183:in `block in convert_file'
        38: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/convert.rb:118:in `convert'
        37: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/document.rb:951:in `convert'
        36: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/converter.rb:388:in `convert'
        35: from /var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/lib/asciidoctor_fb2.rb:75:in `convert_document'
        34: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/document.rb:1020:in `content'
        33: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `content'
        32: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `map'
        31: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `block in content'
        30: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:75:in `convert'
        29: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/converter.rb:388:in `convert'
        28: from /var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/lib/asciidoctor_fb2.rb:103:in `convert_section'
        27: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `content'
        26: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `map'
        25: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `block in content'
        24: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:75:in `convert'
        23: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/converter.rb:388:in `convert'
        22: from /var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/lib/asciidoctor_fb2.rb:107:in `convert_section'
        21: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `content'
        20: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `map'
        19: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `block in content'
        18: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:75:in `convert'
        17: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/converter.rb:388:in `convert'
        16: from /var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/lib/asciidoctor_fb2.rb:107:in `convert_section'
        15: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `content'
        14: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `map'
        13: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:84:in `block in content'
        12: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/abstract_block.rb:75:in `convert'
        11: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/converter.rb:388:in `convert'
        10: from /var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/lib/asciidoctor_fb2.rb:115:in `convert_paragraph'
         9: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/block.rb:113:in `content'
         8: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/substitutors.rb:91:in `apply_subs'
         7: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/substitutors.rb:91:in `each'
         6: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/substitutors.rb:102:in `block in apply_subs'
         5: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/substitutors.rb:379:in `sub_macros'
         4: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/substitutors.rb:379:in `gsub'
         3: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/substitutors.rb:396:in `block in sub_macros'
         2: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/inline.rb:32:in `convert'
         1: from /var/lib/gems/2.5.0/gems/asciidoctor-2.0.10/lib/asciidoctor/converter.rb:388:in `convert'
/var/lib/gems/2.5.0/gems/asciidoctor-fb2-0.2.2/lib/asciidoctor_fb2.rb:193:in `convert_inline_menu': undefined method `empty?' for nil:NilClass (NoMethodError)

```

**Comment by slonopotamus** [comment]
Thanks.

Minimal testcase is:

```adoc
= Title
:experimental:

menu:File[]
```

**Comment by slonopotamus** [comment]
Fixed in 0.2.3 :)

**Comment by dismine** [comment]
You did it! It works now.
